src/TBuffer.h

```diff
@@ -95,7 +96,12 @@ struct TMxpElement
 
 class TBuffer
 {
-    static const QMap<QString, QVector<QChar>> csmEncodingTable; // private!
+    // need to use tr() on encoding names in csmEncodingTable
+    Q_DECLARE_TR_FUNCTIONS(TBuffer)
+
+    // private - a map of computer-friendly encoding names as keys,
+    // values are a pair of human-friendly name + encoding data
+    static const QMap<QString, QPair<QString, QVector<QChar>>> csmEncodingTable;
```

Comment:
💭  Although bundling the encoding data and the "friendly" name as a `QPair` encapsulates the data nicely, from a coding point of view it may be simpler to have two separate `QMaps`...?